mesonbuild/compilers/detect.py

```diff
@@ -246,6 +246,17 @@ def detect_static_linker(env: 'Environment', compiler: Compiler) -> StaticLinker
                 return linkers.MetrowerksStaticLinkerARM(linker)
             else:
                 return linkers.MetrowerksStaticLinkerEmbeddedPowerPC(linker)
+        if 'TASKING VX-toolset' in err:
+            if 'TriCore' in err:
+                return linkers.TaskingTricoreStaticLinker(linker)
+            if 'ARM' in err:
+                return linkers.TaskingARMStaticLinker(linker)
+            if '8051' in err:
+                return linkers.Tasking8051StaticLinker(linker)
+            if 'PCP' in err:
+                return linkers.TaskingPCPStaticLinker(linker)
+            if 'MCS' in err:
+                return linkers.TaskingMCSStaticLinker(linker)
```

Comment:
Do you actually want to fall through here and end up with the system linker, `link.exe` or `ar`?